allennlp/tests/tools/drop_eval_test.py

```diff
@@ -58,10 +75,12 @@ def test_simple_overlap_in_incorrect_cases(self):
 
     def test_multi_span_overlap_in_incorrect_cases(self):
         # only consider bags with matching numbers if they are present
+        # F1 scores of:     1.0        2/3   0.0   0.0   0.0   0.0
```

Comment:
The last one is zero because of a number mismatch?

Reply:
yes